flixel/FlxObject.hx

```diff
@@ -572,6 +572,10 @@ class FlxObject extends FlxBasic
 		flixelType = OBJECT;
 		last = FlxPoint.get(x, y);
 		scrollFactor = FlxPoint.get(1, 1);
+		if (FlxObject.defaultPixelPerfectPosition == null)
```

Comment:
Initializing this static var from an instance method seems hacky. We could just do that when we set the render flags within `FlxG`. No need to turn it into a `Null<Bool>` that way either.